apps/trustlab/src/pages/[[...slugs]].page.js

```diff
@@ -19,7 +21,6 @@ function Page({ blocks, fallback }) {
   }
   return (
     <PageComponent {...pageComponentProps}>
-      <RefreshRouteOnSave />
```

Comment:
@koechkevin Is this not required for Payload Live Preview to work as expected? 
We can maybe add it it the [Page Component](https://github.com/CodeForAfrica/ui/blob/a87b7917f874258a8696fbce37d08509f07198b8/apps/trustlab/src/components/Page/Page.js)

Reply:
There's an issue here @kelvinkipruto, there's a recursive refresh caused by this component.